apps/mpas/README.md

```diff
@@ -69,7 +69,8 @@ source .env/bin/activate
 Install AWS ParallelCluster
 
 ```bash
-pip3 install aws-parallelcluster==2.11.3
+pip3 install pyrsistent==0.16.1
```

Comment:
What is the package use for ?

Reply:
It's a dependency of `aws-parallelcluster`, by default it tries to pull in the latest (0.18.1), which fails with a version mismatch.

OK, scratch that. It works and is not required. Let me update the PR.